pubsub-service-api.yml

```diff
@@ -457,33 +468,35 @@ components:
       format: int32
       minimum: 0
       example: 7
+    MessageProperty:
+      type: object
+      required:
+        - from
+        - data
+      properties:
+        data:
+          description: "The content encoded with base64 standard alphabet with padding."
+          type: string
+          example: "YWJj"
+        from:
+          description: "The peer ID of the sender."
+          type: string
+          example: "QmcXhKGEAJT9DwwzKeEZsWrYTTGkzBdgeKif2MdAXBswJv"
+        pubkey:
+          description: "The sender's public key encoded with base64 standard alphabet with padding in X.509 DER format."
```

Comment:
this should likely be optional for when we can extract the pubkey from the peerID (`from`) to mirror the read part.

Reply:
ho, it IS optional already ... detail in the description in which situations you want to have a pubkey and where not?